hs/t/n/map-dset.txt

```diff
@@ -7,14 +7,27 @@ Verification failed:
   msg: "m[A] is Some(ap)"
   at ./map-dset.rsh:29:10:application
 
-  // Violation witness
-  const address_Alice = Address!val!0;
-  //    ^ from builtin at ./map-dset.rsh:4:30:application
-  const a/27 = 1;
-  //    ^ from evaluating interact("Alice")."get"() at ./map-dset.rsh:16:51:application
-
-  // Theorem formalization
-  require(digest(map0[address_Alice]) == digest(a/27));
+  // Violation Witness
+
+  const a/27 = "Alice".interact.get();
+  //    ^ could = 1
+  //      from: ./map-dset.rsh:16:51:application
+  const b/36 = "Bob".interact.get();
+  //    ^ could = 0
+  //      from: ./map-dset.rsh:21:51:application
+
+
+  // Theorem Formalization
+
+  const map0_0/57 = new Map();
+
```

Comment:
Where do these extra lines come from?

Reply:
For maps and null values, I don't currently show what the values would be. The extra line is a poorly placed `hardline` in my output that I'll fix. I think I'll change this to show `would be null` and `would be <map>` for such values